documentation-site/pages/blog/style-updates/index.mdx

```diff
@@ -0,0 +1,373 @@
+import {Block} from 'baseui/block';
+import {Table} from 'baseui/table';
+
+import metadata from './metadata.json';
+import Layout from '../../../components/layout';
+import {BlogImage, Meta, Caption} from '../../../components/blog';
+
+export default Layout;
+
+<Meta data={metadata} />
+
+<BlogImage
+  src="/static/images/blog/style-updates/baseblack.png"
+  alt="updated base web components with black highlights"
+  caption="Base Web has an updated look"
+/>
+
+Version `8.17` brings some noticeable style updates to Base Web. Read below to find out what has changed and how you can safetly incorporate these changes into your codebase.
+
+## Base Color Palette
+
+The base color primitives for both the default light and dark theme have been adjusted. These have been changed to improve the contrast ratios across our color palette.
+
+The changes in most of the color ways are quite subtle, but the gray (mono) palette has shifted more noticeably. Here are the updated gray scale values:
+
+### Light Theme Changes
+
+<Table
+  columns={['Primitive', 'Old', 'New']}
+  data={[
+    ['mono50', '-', '#F6F6F6'],
```

Comment:
may be an overkill, but I wonder, if we could add a small rectangle next o the color codes, so it's easier to understand?